build.gradle

```diff
@@ -98,12 +98,11 @@ sourceCompatibility = JavaVersion.VERSION_11
 targetCompatibility = JavaVersion.VERSION_11
 
 run {
-    if (S_ARGS != null) {
-        args S_ARGS.split()
-    } else if (System.getProperty("exec.args") != null) {
+    if (System.getProperty("exec.args") != null) {
         args System.getProperty("exec.args").split()
```

Comment:
I tested with this input:

SQL="select * from public.demo_health;"
./gradlew :shuttle:run -Dexec.args="--create demo@openlattice.com --flight ../scripts/flights/demohealth.yaml --config ../scripts/flights/integration.yaml --environment LOCAL --datasource demodata --sql \"$SQL\" --token $TOKEN"
[..., '--datasource', 'demodata', '--sql', '"select', '*', 'from', 'public.demo_health', ...]
I made it work with

    if (System.getProperty("exec.args") != null) {
        Pattern pat = Pattern.compile("[^\\s\"']+|\"([^\"]*)\"|'([^']*)'");
        List<String> execArgs = new ArrayList<String> ();
        MatchResult matcher = pat.matcher(System.getProperty("exec.args"));
        while (((Matcher)matcher).find()){
            execArgs.add(matcher.group(0))
        }
        args execArgs
    } else if (S_ARGS != null) {
        args S_ARGS.split()
    } else if (project.hasProperty('appArgs')) {
        args Eval.me(appArgs)
    }
Not sure why this was never a problem with arguments passed with IntelliJ or eg. compiled command line.